### ISSUE TITLE
Manual changes to /etc/hosts are reverted on DHCP > Save Changes

### ISSUE THREAD

**Issue by bbecker-inlogik** [ericpaulbishop/gargoyle#1009]
Steps to reproduce issue:
* manually edit the /etc/hosts file via ssh.
* Via web front end Connection > DHCP > Save Changes
* manual changes to /etc/hosts are reverted

The createHostCommands variable should be removed completely, since DHCP configuration no longer lives in /etc/hosts there is no need to revert this file to factory setting.  
https://github.com/ericpaulbishop/gargoyle/blob/ae547ce5113ce1c6e260112bcdcab9f41e3e7ab8/package/gargoyle/files/www/js/dhcp.js#L105

**Comment by lantis1008** [comment]
Thanks for reporting, I will consider a fix.
We do place local host information in there intentionally so there is still a requirement.

**Comment by bbecker-inlogik** [comment]
Perhaps rather than clearing the file, it could append the localhost lines only if they are are missing.

**Comment by eode** [comment]
Is there anyway to add hosts to the /etc/hosts file, or otherwise add hosts to be served up via DNS, without knowing a device mac as the UI requires?

**Comment by lantis1008** [comment]
That is correct by the current logic. It removes /etc/ethers and /etc/hosts on save.
It also removes all sections of type “host” in /etc/config/dhcp and recreates from the GUI.

It was not designed to be modified otherwise. 
It could be changed to leave the other files alone if needed, but the change to config is probably needed.
